eng/scripts/Language-Settings.ps1

```diff
@@ -405,11 +405,17 @@ function EnsureCustomSource($package) {
   }
 
   Write-Host "Checking custom package source for $($package.Name)"
-  $existingVersions = Find-Package `
-    -Name $package.Name `
-    -Source CustomPackageSource `
-    -AllVersions `
-    -AllowPrereleaseVersions
+  try {
```

Comment:
Some of the management track 1 packages do not exist in public feeds anymore. 
<img width="438" alt="image" src="https://user-images.githubusercontent.com/48036328/198365441-4597f2e7-4a00-491c-a432-184d67f799b9.png">

Reply:
Use the try catch to skip the failures

Reply:
I suspect they may have never been in our devops feed. If we need namespaces for them we might need to either add them or use nuget.org.

Reply:
It doesn't affect main/live site. Will add some fallback logic here to make sure we download the right package.